_includes/linetodayPlayer.html

```diff
@@ -0,0 +1,13 @@
+<!-- Feel free to change the width and height to your desired video size. -->
+<!-- Remember to include 2-letter country name, too! (LINE TODAY is available on different regions) -->
```

Comment:
This looks good, could you add in the various possible options for the 2-letter country names? I believe the possible countries are Taiwan, Thailand, Indonesia, and Hong Kong. Thanks!